src/libcommonserver/io/iohelper_win.cpp

```diff
@@ -81,6 +81,8 @@ IoError ntStatus2ioError(NTSTATUS status) noexcept {
         case STATUS_SUCCESS:
             return IoError::Success;
         case STATUS_ACCESS_DENIED:
+        case ERROR_SHARING_VIOLATION:
+        case ERROR_LOCK_VIOLATION:
```

Comment:
Is there a simple unit test that can be written to prevent a regression?

Reply:
Added here: [0a15aaa](https://github.com/Infomaniak/desktop-kDrive/pull/381/commits/0a15aaa25e18c116700d456204e903293791741f)

Reply:
It look like this test isn't working on mac and linux.
I have to find a way to lock the file on all OS.

Reply:
Done